bootstrap.sh

```diff
@@ -280,29 +289,36 @@ install_homebrew() {
 
   # Download Homebrew.
   export GIT_DIR="$HOMEBREW_REPOSITORY/.git" GIT_WORK_TREE="$HOMEBREW_REPOSITORY"
-  # shellcheck disable=SC2086
   git init $Q
   git config remote.origin.url "https://github.com/Homebrew/brew"
   git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-  # shellcheck disable=SC2086
   git fetch $Q --tags --force
-  # shellcheck disable=SC2086
   git reset $Q --hard origin/master
   unset GIT_DIR GIT_WORK_TREE
   logk
 
   # Update Homebrew.
   export PATH="$HOMEBREW_PREFIX/bin:$PATH"
-  log "Updating Homebrew:"
-  brew update -q
+  logn "Updating Homebrew:"
+  brew update --quiet
   logk
 
+  # On Apple M1 machines we need to add this to the profile
+  if [[ "$(uname -m)" == "arm64" ]]; then
```

Comment:
This block I added by looking at brew's install code.

For now, let's take this PR but I would like to investigate why the brew install script did not work at first.